app/src/main/res/values/strings.xml

```diff
@@ -812,5 +814,6 @@ Please report any problems you encounter via Github.</string>
     <string name="browse">Browse</string>
     <string name="no_sub_folder_is_selected">No sub folder is selected</string>
     <string name="share_device_id_chooser">Share device ID with</string>
+    <string name="title_activity_themed_app_compat">ThemedAppCompatActivity</string>
```

Comment:
Seems like a weird string to translate?

Reply:
That is a really good point. It looks like this is another one of those things that was added automatically by android studio when the ThemedAppCompatActivity class was created. It can be removed.

Reply:
This has been removed.